devicetypes/smartthings/zigbee-multi-switch.src/zigbee-multi-switch.groovy

```diff
@@ -277,23 +289,31 @@ private getChildCount() {
 		case "HS6SW3A-W-EF-3.0":
 			return 3
 		case "E220-KR4N0Z0-HA":
+	case "E220-KR4N0Z1-HA":
+	case "E220-KR4N0Z2-HA":
 		case "ZB-SW04":
 		case "JZ-ZB-004":
 		case "SBM300Z4":
 			return 4
 		case "E220-KR5N0Z0-HA":
+	case "E220-KR5N0Z1-HA":
+	case "E220-KR5N0Z2-HA":
 		case "ZB-SW05":
 		case "JZ-ZB-005":
 		case "SBM300Z5":
 			return 5
 		case "E220-KR6N0Z0-HA":
+	case "E220-KR6N0Z1-HA":
+	case "E220-KR6N0Z2-HA":
 		case "ZB-SW06":
 		case "JZ-ZB-006":
 		case "SBM300Z6":
 			return 6
 		case "E220-KR2N0Z0-HA":
+	case "E220-KR2N0Z1-HA":
+	case "E220-KR2N0Z2-HA":
```

Comment:
Please add one more tab to Your `case ` statements so that they would align with others.